File: PT5.java
import java.util.Scanner;

public class PT5 {
    public static Scanner input = new Scanner(System.in);
    public static int num1, num2;

    public static void main(String[] args) {
        System.out.println("Choose Operation");
        System.out.println("[1] Addition");
        System.out.println("[2] Subtraction");
        System.out.println("[3] Multiplication");
        System.out.println("[4] Division");
        System.out.print("Choice: ");
        int choice = input.nextInt();
        switch (choice) {
            case 1:
                addition();
                break;
            case 2:
                subtraction();
                break;
            case 3:
                multiplication();
                break;
            case 4:
                division();
                break;
            default:
                System.out.println("Invalid Choice. Good Byte!");
        }
    }

    public static void addition() {
        System.out.print("Enter 1st number: ");
        num1 = input.nextInt();
        System.out.print("Enter 2nd number: ");
        num2 = input.nextInt();
        System.out.println("The result of " + num1 + " and " + num2 + " is " + performOperation(num1, num2, '+'));
    }

    public static void subtraction() {
        System.out.print("Enter 1st number: ");
        num1 = input.nextInt();
        System.out.print("Enter 2nd number: ");
        num2 = input.nextInt();
        System.out.println("The result of " + num1 + " and " + num2 + " is " + performOperation(num1, num2, '-'));
    }

    public static void multiplication() {
        System.out.print("Enter 1st number: ");
        num1 = input.nextInt();
        System.out.print("Enter 2nd number: ");
        num2 = input.nextInt();
        System.out.println("The result of " + num1 + " and " + num2 + " is " + performOperation(num1, num2, '*'));
    }

    public static void division() {
        System.out.print("Enter 1st number: ");
        num1 = input.nextInt();
        System.out.print("Enter 2nd number: ");
        num2 = input.nextInt();
        System.out.println("The result of " + num1 + " and " + num2 + " is " + performOperation(num1, num2, '/'));
    }

    public static int performOperation(int num1, int num2, char operation) {
        int result;
        switch (operation) {
            case '+':
                result = num1 + num2;
                break;
            case '-':
                result = num1 - num2;
                break;
            case '*':
                result = num1 * num2;
                break;
            case '/':
                result = num1 / num2;
                break;
            default:
                result = 0;
        }
        return result;
    }
}

File: PT5.cs
using System;

namespace IntroToCSG11 //<-- Name of your solution/project
{
    internal static class Program
    {
        public static void Main()
        {
            Console.WriteLine("[1] Addition");
            Console.WriteLine("[2] Subtraction");
            Console.WriteLine("[3] Multiplication");
            Console.WriteLine("[4] Division");
            Console.Write("Enter Choice: ");
            var choice = Console.ReadLine();
            switch (Convert.ToInt32(choice))
            {
                case 1:
                    Addition();
                    break;
                case 2:
                    Subtraction();
                    break;
                case 3:
                    Multiplication();
                    break;
                case 4:
                    Division();
                    break;
                default:
                    Console.WriteLine("Invalid Choice. Good Bye!");
                    break;
            }
            Console.ReadLine();
        }

        private static void Addition()
        {
            Console.Write("Enter 1st Number: ");
            var num1 = Console.ReadLine();
            Console.Write("Enter 2nd Number: ");
            var num2 = Console.ReadLine();
            Console.WriteLine("The result of " + num1 + " and " + num2 + " is " + PerformOperation(int.Parse(num1), int.Parse(num2), '+'));
        }
        
        private static void Subtraction()
        {
            Console.Write("Enter 1st Number: ");
            var num1 = Console.ReadLine();
            Console.Write("Enter 2nd Number: ");
            var num2 = Console.ReadLine();
            Console.WriteLine("The result of " + num1 + " and " + num2 + " is " + PerformOperation(int.Parse(num1), int.Parse(num2), '-'));
        }
        
        private static void Multiplication()
        {
            Console.Write("Enter 1st Number: ");
            var num1 = Console.ReadLine();
            Console.Write("Enter 2nd Number: ");
            var num2 = Console.ReadLine();
            Console.WriteLine("The result of " + num1 + " and " + num2 + " is " + PerformOperation(int.Parse(num1), int.Parse(num2), '*'));
        }
        
        private static void Division()
        {
            Console.Write("Enter 1st Number: ");
            var num1 = Console.ReadLine();
            Console.Write("Enter 2nd Number: ");
            var num2 = Console.ReadLine();
            Console.WriteLine("The result of " + num1 + " and " + num2 + " is " + PerformOperation(int.Parse(num1), int.Parse(num2), '/'));
        }

        private static int PerformOperation(int num1, int num2, char operation)
        {
            var result = 0;
            if (operation == '+')
                result = num1 + num2;
            else if (operation == '-')
                result = num1 - num2;
            else if (operation == '*')
                result = num1 * num2;
            else if (operation == '/') result = num1 / num2;
            
            return result;
        }
    }
}
